packages/options/legacy/class-jetpack-options.php

```diff
@@ -167,14 +167,27 @@ public static function is_network_option( $option_name ) {
 	}
 
 	/**
-	 * Returns the requested option.  Looks in jetpack_options or jetpack_$name as appropriate.
+	 * Filters the requested option.
+	 * This is a wrapper around `get_option_from_database` so that we can filter the option.
 	 *
 	 * @param string $name Option name. It must come _without_ `jetpack_%` prefix. The method will prefix the option name.
 	 * @param mixed  $default (optional).
 	 *
 	 * @return mixed
 	 */
 	public static function get_option( $name, $default = false ) {
+		return apply_filters( 'jetpack_options', self::get_option_from_database( $name, $default ), $name );
```

Comment:
Could you add a docblock for that new filter and its parameters?

Reply:
Done :)